conversion.go

```diff
@@ -565,7 +566,25 @@ func (dst *Int) Scan(src interface{}) error {
 	}
 	switch src := src.(type) {
 	case string:
-		return dst.SetFromDecimal(src)
+		splt := strings.SplitN(src, "e", 2)
+		if len(splt) < 2 {
+			return dst.SetFromDecimal(src)
+		}
+		err := dst.SetFromDecimal(splt[0])
+		if err != nil {
+			return err
+		}
+		if splt[1] == "0" {
+			return nil
+		}
```

Comment:
`1e0` should be `1`, not `0` ... ? 
Not sure what `0e0` is defined as, need to check that.

Reply:
isn't that what this does?

In the previous lines, i do dst.SetFromDecimal(splt[0])

then if `splt[1] == 0`, then i just return, since the value of the string is just the part in front of the "e"

Reply:
My bad, yes you're right